huggingface_pytorch-pretrained-bert_bert.md

```diff
@@ -59,22 +62,31 @@ segments_ids = [0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1]
 segments_tensors = torch.tensor([segments_ids])
 tokens_tensor = torch.tensor([indexed_tokens])
 
+model = torch.hub.load('huggingface/pytorch-pretrained-BERT', 'bertModel', 'bert-base-cased', force_reload=False)
+model.eval()
+
 with torch.no_grad():
     encoded_layers, _ = model(tokens_tensor, segments_tensors)
 ```
 
 We can predict masked tokens using `bertForMaskedLM`.
 
 ```python
-model = torch.hub.load('huggingface/pytorch-pretrained-BERT', 'bertForMaskedLM', 'bert-base-cased', force_reload=False)
-model.eval()
+import torch
+tokenizer = torch.hub.load('huggingface/pytorch-pretrained-BERT', 'bertTokenizer', 'bert-base-cased', do_basic_tokenize=False, force_reload=False)
```

Comment:
i think the `force_reload=False` can be removed from all of the calls right...

Reply:
Done in cb1747b5f58935b77a1d779b2ce16190ea483d1e !